documentation/modules/oauth/con-oauth-authentication-flow.adoc

```diff
@@ -65,29 +81,14 @@ When used with the OAuth 2.0 over PLAIN callbacks, Kafka clients authenticate wi
 
 * A long-lived access token, obtained manually at configuration time
 
-The client must be enabled to use PLAIN authentication, and provide a `username` and `password`.
-If the password is prefixed with `$accessToken:` followed by the value of the access token, the Kafka broker will interpret the password as the access token.
+For both methods, the client must provide a `username` and `password` to pass credentials to the Kafka broker.
+If the password is prefixed with `$accessToken:` followed by the value of an access token, the Kafka broker will interpret the password as the access token.
```

Comment:
```suggestion
When listener is configured with `tokenEndpointUri`, if the password is prefixed with `$accessToken:` followed by the value of an access token, the Kafka broker will interpret the password as the access token.
```

Reply:
Tried to break a very long sentence up a bit.

> For both methods, the client must provide the PLAIN properties `username` and `password` to pass credentials to the Kafka broker.
> For the Kafka broker to interpret the password as an access token, it requires an `$accessToken:` prefix followed by the value of the access token. The listener configuration must have a token endpoint (`tokenEndpointUri`) configured for authentication. 
> Otherwise, the Kafka broker will interpret the `username` as the client ID and the `password` as the client secret.
>